Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    File.exist?("~/Applications/Firefox.app")
+  end
```

Comment:
The Cask defaults to symlinking into ` ~/Applications ` which is why I included it mainly, but happy to cut.